src/register-org/containers/registered-address/registered-address.component.html

```diff
@@ -1,10 +1,11 @@
-<main id="content" class="govuk-main-wrapper" role="main" tabindex="-1">
+<main id="content" class="govuk-main-wrapper" role="main" tabindex="-1" #mainContent>
```

Comment:
@connorpgpmcelroy - `#mainContent` is no longer required as scrolling to the top is automatically handled by the `xuilib-hmcts-error-summary` component.

Reply:
Ok, will check that and remove if so